core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1459,6 +1459,13 @@
    */
   public static final String HANDOFF_SIZE = "carbon.streaming.segment.max.size";
 
+  /**
+   * enable auto handoff streaming segment
```

Comment:
Can you describe more how this feature is working, and you can copy the description to document in `/doc` folder

Reply:
ok, I will raise pr to finish streaming doc